app/models/user.rb

```diff
@@ -1,10 +1,14 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
```

Comment:
- [ ] Based on the requirements of this project, this app should ask for email confirmations, hence `:confirmable` should be enabled in the user model and make sure that a confirmation message is sent to a new user after registering to confirm his/her email.